model_compression_toolkit/gptq/keras/gptq_training.py

```diff
@@ -169,16 +162,17 @@ def get_activation_quantizer_holder(self, n: common.BaseNode) -> Union[None, Cal
         Returns:
             A ActivationQuantizationHolder layer for the node activation quantization.
         """
-        if self._is_gptq_applicable(n): # If it was wrapped before - add holder instead
-            _, activation_quantizers = quantization_builder(n, self.gptq_config)
-            # Holder by defenition uses a single quantizer for the activation quantization
-            # thus we make sure this is the only possible case (unless it's a node we no activation
-            # quantization, which in this case has an empty list).
-            if len(activation_quantizers) > 0:
-                assert len(activation_quantizers) == 1, f'ActivationQuantizationHolder supports a ' \
-                                                        f'single quantizer but {len(activation_quantizers)} quantizers were found for node {n}'
-                return ActivationQuantizationHolder(activation_quantizers[0])
-        return None
+        _, activation_quantizers = quantization_builder(n, self.gptq_config)
```

Comment:
Just a suggestion (need to think about that) - we actually call quantization_builder twice for each node (here and in gptq_wrapper) but each time only take one out of the weights/activation quantizers.
Can we reduce it to a single call?
Maybe call it in the __init__ method of the class and save a mapping between the node to each type of quantizers?